app/src/main/java/org/mozilla/focus/browser/LocalizedContent.java

```diff
@@ -69,7 +69,7 @@ private static void loadAbout(@NonNull final WebView webView) {
 
         final String data = HtmlLoader.loadResourceFile(context, R.raw.about, substitutionMap);
         // We use a file:/// base URL so that we have the right origin to load file:/// css and image resources.
-        webView.loadDataWithBaseURL("file:///android_res/raw/about.html", data, "text/html", "UTF-8", null);
+        webView.loadDataWithBaseURL(URL_ABOUT, data, "text/html", "UTF-8", URL_ABOUT);
```

Comment:
This works as long as those pages do not load or link to any other resources in the raw/ folder.